sdk/ai/azure-ai-resources/azure/ai/resources/_index/_dataindex/entities/_builders/data_index_func.py

```diff
@@ -736,4 +736,5 @@ def _resolve_connection_id(ml_client, connection: Optional[Union[str, WorkspaceC
         # Handle azure.ai.resources Connections
         connection = connection._workspace_connection
 
+    print(connection.id) # REMOVE
```

Comment:
It seems like we can remove this debugging line?

Reply:
Yes! Sorry I missed this line